app/src/app.tsx

```diff
@@ -13,6 +13,7 @@ import { GlobalsProvider, IGlobals } from './globalsContext';
 import { PageDataProvider } from './PageDataContext';
 import { HomePage } from './pages/HomePage';
 import { getHomePageData } from './pages/HomePage/getHomePageData';
+import { UserPage } from './pages/UserPage/UserPage';
```

Comment:
since it has account in the url path, you should call this AccountPage instead of UserPage (change everywhere)